### PR TITLE
feat(back): github create issue reaction

### PR DESCRIPTION
**Pull request by EdenComp**

### CODE REVIEW

File: backend/workers/github/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Typescript project template
```
Comment (RezaRahemtola) [inline backend/workers/github/README.md:1]: you can remove this :rofl:

File: backend/workers/github/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "github",
```
Comment (RezaRahemtola) [inline backend/workers/github/package.json:2]:
```suggestion
  "name": "github-area-worker",
```

File: backend/workers/github/src/issues/create.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import {z} from "zod";
+import {Octokit, RequestError} from "octokit";
+import {credentials} from "@grpc/grpc-js";
+import parseArguments, {GithubAuthSchema} from "../util/params";
+import {AreaBackServiceClient} from "../proto/area_back";
+import "../proto/google/protobuf/struct";
+import {GithubErrorData} from "../util/types";
+import {onError, onReaction} from "../util/grpc";
```
Comment (RezaRahemtola) [inline backend/workers/github/src/issues/create.ts:8]: doesn't look linted to me but doesn't really matter

**Review by RezaRahemtola — CHANGES_REQUESTED** [review]
I suppose you tested locally that it was working ?

**Comment by EdenComp** [comment]
> I suppose you tested locally that it was working ?

ofc

File: backend/workers/github/README.md (hunk 1 above)
Comment (EdenComp) [reply backend/workers/github/README.md:1]: oupsie

File: backend/workers/github/src/issues/create.ts (hunk 3 above)
Comment (EdenComp) [reply backend/workers/github/src/issues/create.ts]: You're right! The eslint config didn't trigger prettier

**Review by RezaRahemtola — APPROVED** [review]

**Pull request merged**